chromium_src/components/search_engines/android/template_url_service_android.cc

```diff
@@ -20,3 +23,59 @@ jboolean TemplateUrlServiceAndroid::DoesDefaultSearchEngineHaveLogo(
 
   return DoesDefaultSearchEngineHaveLogo_ChromiumImpl(env, obj);
 }
+
+jboolean TemplateUrlServiceAndroid::AddSearchEngine(
```

Comment:
Please do not change naming conventions by adding `SearchEngine`. You are calling `Add` so that's what this method should be called

Reply:
Resolved here : https://github.com/brave/brave-core/pull/27768/commits/7915f91d3c9c8022dca7208211ef018551c0c147